src/Symfony/Component/HttpFoundation/Session/Storage/Handler/RedisSessionHandler.php

```diff
@@ -0,0 +1,116 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\HttpFoundation\Session\Storage\Handler;
+
+/**
+ * Redis based session storage handler based on the Redis class
+ * provided by the PHP redis extension.
+ *
+ * Direct port of MemcachedSessionHandler to Redis.
+ *
+ * @see http://php.net/redis
+ *
+ * @author Drak <drak@zikula.org>
+ * @author Dalibor Karlović <dkarlovi@gmail.com>
+ */
+class RedisSessionHandler extends AbstractSessionHandler
+{
+    /**
+     * @var \Redis
```

Comment:
this docblock should be removed: the constructor already has the info

Reply:
It won't when we start accepting multiple instances.

Reply:
it will thanks to the docblock on the constructor, which phpstorm will read, isn't it?

Reply:
You're right, removing.